examples/data-router/src/app.tsx

```diff
@@ -44,10 +44,40 @@ let router = createBrowserRouter(
         loader={deferredLoader}
         element={<DeferredPage />}
       />
+      <Route
+        path="lazy"
+        lazy={async () => {
+          console.log("loading lazy");
+          await sleep(1000);
+          console.log("done loading lazy");
+          let {
+            default: Component,
+            loader,
+            action,
+            ErrorBoundary,
+            shouldRevalidate,
+          } = await import("./lazy");
+
+          return {
+            element: <Component />,
+            loader,
+            action,
+            shouldRevalidate,
+            ...(ErrorBoundary
+              ? {
+                  errorElement: <ErrorBoundary />,
+                  hasErrorBoundary: true,
+                }
+              : {}),
+          };
+        }}
+      />
     </Route>
   )
 );
 
+window.router = router;
```

Comment:
Only temporary - useful to check `window.router.routes` before and after lazily loading modules